### PR TITLE
deploy to base goerli

### PR DESCRIPTION
**Pull request by JFrankfurt**

### CODE REVIEW

File: script/deployParameters/DeployBASEGoerli.s.sol [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import {DeployUniversalRouter} from '../DeployUniversalRouter.s.sol';
+import {RouterParameters} from 'contracts/base/RouterImmutables.sol';
+
+contract DeployBASEGoerli is DeployUniversalRouter {
+    function setUp() public override {
+        params = RouterParameters({
+            permit2: 0x000000000022D473030F116dDEE9F6B43aC78BA3,
+            weth9: 0x44D627f900da8AdaC7561bD73aA745F132450798,
+            seaportV1_5: UNSUPPORTED_PROTOCOL,
```
Comment (hensha256) [inline script/deployParameters/DeployBASEGoerli.s.sol:12]: isnt unsupported - should be `0x00000000000000ADc04C56Bf30aC9d3c0aAF14dC`

File: script/deployParameters/DeployBASEGoerli.s.sol [hunk 2]
```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+import {DeployUniversalRouter} from '../DeployUniversalRouter.s.sol';
+import {RouterParameters} from 'contracts/base/RouterImmutables.sol';
+
+contract DeployBASEGoerli is DeployUniversalRouter {
```
Comment (hensha256) [inline script/deployParameters/DeployBASEGoerli.s.sol:7]: In their docs, blogs etc its just Base not BASE

**Review by hensha256 — CHANGES_REQUESTED** [review]
Needs to support seaport too 🙏

**Comment by JFrankfurt** [comment]
this adds goerli base support and we should include it

File: script/deployParameters/DeployBASEGoerli.s.sol
Comment (ewilz) [inline script/deployParameters/DeployBASEGoerli.s.sol:1]: `DeployBaseGoerli.s.sol` ?

File: deploy-addresses/BASE-goerli.json
Comment (ewilz) [inline deploy-addresses/BASE-goerli.json:1]: `base-goerli.json` ?

**Pull request merged**